src/main.c

```diff
@@ -1778,7 +1778,14 @@ static void cmdline_handler(int argc, char **argv)
 		{CONVERT, "convert", "convert <INBUNDLE> <OUTBUNDLE>", "Convert to casync index bundle and store", convert_start, convert_group, FALSE},
 		{CHECKSUM, "checksum", "checksum <DIRECTORY>", "Deprecated", checksum_start, NULL, FALSE},
 		{INFO, "info", "info <FILE>", "Print bundle info", info_start, info_group, FALSE},
-		{STATUS, "status", "status", "Show system status", status_start, status_group, TRUE},
+		{STATUS, "status", "status",
+			"Show system status\n\n"
+			"List of status commands: (default image is the currently booted image)\n"
+			" mark-good [booted | other | <SLOT_NAME>] \tMark the image as good \n"
+			" mark-bad [booted | other | <SLOT_NAME>] \tMark the image as bad\n"
+			" mark-active [booted | other | <SLOT_NAME>] \tMark the image as active\n",
```

Comment:
The last line break is not required here, as this block will be printed with a line break, anyway.